docs/interactions/transactions.md

```diff
@@ -322,6 +322,10 @@ following aspects of a transaction to be configured:
     confirmation at the application level to avoid inconsistencies. USE
     WITH GREAT CARE!
 
+-   `skipDBCacheRead(boolean)` - Enables/disables JanusGraph database level 
+    cache access during read operations. Doesn't have any effect if database 
+    level cache was disabled during JanusGraph instance creation.
```

Comment:
(nitpick): `JanusGraph instance creation` might confuse some users - "instance creation" sounds like the time when the graph is first created. What about "if database level cache was disabled via config `cache.db-cache`"?